src/iceberg/executor/seals.py

```diff
@@ -6,6 +6,8 @@
 
 from __future__ import print_function
 import os
+import pandas as pd
```

Comment:
Do we use Pandas only for reading a csv file? If so, should we use something lighter like https://docs.python.org/3/library/csv.html? Pandas takes a fair amount of time downloading and installing.

Reply:
@mturilli I made the requested change and it works for me.